libs/blocks/pdf-viewer/pdf-viewer.js

```diff
@@ -44,7 +44,7 @@ const init = async (a) => {
   const handleViewSdkReady = () => {
     const adobeDCView = new AdobeDC.View(
       {
-        clientId,
+        clientId: '1d9058b1b677477a8cf1d3a030f71f1e',
```

Comment:
NOTE: This hardcoded client id is necessary for testing. I'll remove the changes after it's been verified